inbox/mailsync/backends/imap/common.py

```diff
@@ -271,10 +272,10 @@ def create_imap_message(
     )
     new_message = Message.create_from_synced(
         account=account,
-        mid=raw_message.uid,
+        imap_uid=raw_message.uid,
```

Comment:
Changed this to `imap_uid` because I don't know what `mid` means without reading code.

Is this `middle` like `mid-way` `mid-journey`? No
Is this message id like `Message-ID` header? No
Is this the unique id that IMAP uses to identify a message inside a folder, yes.